File: Testing/APT Parsing/NASR-ParseAllAPT.cpp
#include <fstream>
#include <iostream>
#include <sstream>
#include <vector>

void PrintAirport (const std::string input)
{
}

int main (int argc, char *argv[]) 
{	
	std::ifstream infile ("APT.txt");
	std::ofstream outFile;
	
	outFile.open ("airports.txt");
	
	std::string line;
	
	while (std::getline (infile, line))
	{	
		std::istringstream iss (line);
		
		std::string input = iss.str();
		std::string type = input.substr(0,3);
			
		if (0 == type.compare("APT"))
		{
			std::string airport = input.substr(27,4);
			std::string latitude = input.substr(538,12);
			std::string longitude = input.substr(565,12);
			
			std::cout << airport << ": " << latitude << ", " << longitude << std::endl;
			outFile << airport << ": " << longitude << ", " << latitude << std::endl;
		}
	}
	
	outFile.close();
}

File: Prototyping/7-Export All Aiports To CSV-Python/NASR-ParseAll-APT.py
out_file = open("output-airports.txt", 'w')

airport_file = open('APT-Sample.txt', 'r')
text = airport_file.read()
lines = text.split("\n")

for line in lines:

	type = line[0:3]
		
	if (type == "APT"):
		airport = line[27:30]
		latitude = line[538:549]
		longitude = line[565:576]
		
		final_string = "%s: %sN, %sW\n" % (airport, latitude, longitude)
		
		print(final_string)
		out_file.write(final_string)
		
out_file.close()
